# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/snes9x/hotfix-gcc15.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- snes9x/external/glslang/SPIRV/SpvBuilder.h.vanilla	2025-04-29 00:54:48.881241560 +0300
+++ snes9x/external/glslang/SPIRV/SpvBuilder.h	2025-04-29 00:55:24.886043942 +0300
@@ -63,6 +63,7 @@
 #include <stack>
 #include <unordered_map>
 #include <map>
+#include <stdint.h>
 
 namespace spv {
 
